client/my-sites/plans/current-plan/index.jsx

```diff
@@ -133,25 +134,29 @@ class CurrentPlan extends Component {
 					/>
 				) }
 
-				<CurrentPlanHeader
-					selectedSite={ selectedSite }
-					isPlaceholder={ isLoading }
-					title={ title }
-					tagLine={ tagLine }
-					currentPlanSlug={ currentPlanSlug }
-					currentPlan={ currentPlan }
-					isExpiring={ isExpiring }
-					isAutomatedTransfer={ isAutomatedTransfer }
-					includePlansLink={ currentPlan && isFreeJetpackPlan( currentPlan ) }
-				/>
-				{ isEnabled( 'jetpack/checklist' ) &&
-					isJetpack &&
-					! isAutomatedTransfer && (
-						<Fragment>
-							<QueryJetpackPlugins siteIds={ [ selectedSiteId ] } />
-							<JetpackChecklist />
-						</Fragment>
-					) }
+				{ this.props.showThankYou ? (
+					<ThankYouHeader
```

Comment:
The more I look at the UI of this component and the more I think about it, the less I consider it a "header". It's a large section and probably shouldn't really be considered a "header", but more a "Plan Setup Success" or "Plan Setup Thank You" component. What do you think?

Reply:
The way the UI / Setup is split (depends on #26886), this component is informative UI, it doesn't have much to do with setup other than displaying messaging while setup occurs.

When setup is complete, there's another card to be shown that has yet to be implemented and I'm not sure exactly how that will function.

Regarding "header" language, it's a header in the sense that it renders at the top of the column. I'm don't have any strong feelings about this, would you prefer it be swapped for something like "ThankYouCard"?

There will be more opportunities to consider this before the change is launched. The "Thank You" part of this Component name may also come to be inaccurate depending on how the UI for after the setup process is complete is implemented:

![complete](https://user-images.githubusercontent.com/841763/45144095-6b2f5700-b1bd-11e8-9ba7-93a420f3700a.png)

Reply:
Thanks for elaborating; and yeah, I think something broader like `ThankYouCard` makes more sense.

Reply:
Annoyance - thank-you-card exists, eslint namespace enforced classes collide 😩

Reply:
👍 16b17705dc2f9b9d1f1775631cae0e77b2c6ab48